os/arch/arm/src/s5j/s5j_pm_idle.c

```diff
@@ -0,0 +1,125 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ *
+ *  arch/arm/src/s5j/s5j_pm_idle_init_data.c
+ *
+ ****************************************************************************/
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+#include <errno.h>
+
+#include <tinyara/config.h>
+#include <tinyara/arch.h>
+#include <tinyara/board.h>
+#include <tinyara/irq.h>
+#include <tinyara/pm/pm.h>
+
+#include "s5j_clock.h"
+#include "s5j_pm.h"
+#include "up_arch.h"
+#include "up_internal.h"
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Private Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: s5j_idle_pmnormal
+ *
+ * Description:
+ *   Enter NORMAL mode.
+ *
+ ****************************************************************************/
+static int s5j_idle_pmnormal(void)
+{
+	errno = OK;
+	s5j_clk_pll_select_mux(true);
+
+	return errno;
+}
+
+/****************************************************************************
+ * Name: s5j_idle_pmstop
+ *
+ * Description:
+ *   Enter STOP mode.
+ *
+ ****************************************************************************/
+static int s5j_idle_pmstandby(void)
+{
+	errno = OK;
+	s5j_clk_pll_select_mux(false);
+
+	return errno;
+}
+
+/****************************************************************************
+ * Name: s5j_idle_pmstandby
+ *
+ * Description:
+ *   Enter STANDBY mode.
+ *
+ * Input Parameters:
+ *   None
+ *
+ ****************************************************************************/
+static int s5j_idle_pmstop(void)
+{
```

Comment:
Comment header name and fucntion name doesn't match

Reply:
ok will modify in main PR